### ISSUE TITLE
Change branch structure of this repository

### ISSUE THREAD

**Issue by thinkh** [lineupjs/lineupjs#196]
I suggest to change the branch structure of this lineupjs repository in the following way:

* _master_: LineUp 3 releases, no direct commits and feature PRs (exception are release PRs)
* _develop_: New branch from the master for the development of LineUp 3
* rename current _develop_ to _lineup-v4_: branch for LineUp 4 development

@sgratzl Would that be ok for you? If yes, I would rename and create the branches.

**Comment by thinkh** [comment]
@sgratzl I going to change the branch structure this week. Please let me know, if you have any comments or thoughts on that. Thanks.

**Comment by sgratzl** [comment]
it depends which version should be your main one. in the v3 case, your suggestion makes sense, in the v4 case it could be like
 * master ... latest (pre) release of v4
 * develop ... v4 dev branch
 * v3-maintenance ... v3 maintenance branch

**Comment by thinkh** [comment]
I discussed this also internally again and we agreed that the last stable release, i.e. LineUp v3, should be available in the `master`. Hence, the development of LineUp v3 is done in the `develop` branch and the development of LineUp v4 is available in the `lineup-v4` branch. I changed the branch structure accordingly and pointed the open PRs to the correct branches again.

Furthermore, I updated the labels in this repository. New are the labels `release: patch`, `release: minor`, and `release: major` that should be attached to each pull request. That will make it easier to figure out the version number for an upcoming release.
